src/mapblock.h

```diff
@@ -73,15 +73,12 @@ class MapBlock
 		m_orphan = true;
 	}
 
-	void reallocate()
-	{
-		for (u32 i = 0; i < nodecount; i++)
-			data[i] = MapNode(CONTENT_IGNORE);
-		raiseModified(MOD_STATE_WRITE_NEEDED, MOD_REASON_REALLOCATE);
```

Comment:
Note that (1) reallocate was only ever called from the constructor, and (2) it did not make sense to mark the block as unclean there.

Reply:
A better name would have been `MOD_REASON_CREATED` (as reallocate was only ever called from the constructor), which would not make much sense (to me anyway). I think this is a vestige from previous changes.

I could rename to `MOD_REASON_CREATED` and raise it from the constructor to be sure to have the old behavior. That seems weird, though.